model/Human-GEM.yml

```diff
@@ -119559,7 +119559,7 @@
           - MAM02689m: 1
       - lower_bound: 0
       - upper_bound: 1000
-      - gene_reaction_rule: "ENSG00000084754 and ENSG00000138029 and ENSG00000167315"
```

Comment:
`MAR03181` has not been mentioned in this [PR](https://github.com/SysBioChalmers/Human-GEM/pull/808) or [issue](https://github.com/SysBioChalmers/Human-GEM/issues/757). The GPR is also different from other reactions. This modification seems inappropriate.

Reply:
oh yea that is definitely wrong; I think I must've misread `MAR03187` (which is on the list) as `MAR03181`. I will revert that change